font.go

```diff
@@ -125,6 +125,7 @@ func getInfoFromTrueType(fileStr string, msgWriter io.Writer, embed bool, encLis
 			uv := encList[j].uv
 			pos, ok := ttf.Chars[uint16(uv)]
 			if ok {
+				//TODO: check why diffrent presentations of same arabic character have equal widths
```

Comment:
```suggestion
				// TODO(hmmftg): check why different presentations of the same arabic character have equal widths
```

but I must say I am not sure to understand what the comment is supposed to convey :)

Reply:
it's been fixed and comment was forgotten sorry